### ISSUE TITLE
`package::function` syntax in `command`

### ISSUE THREAD

**Issue by ashiklom** [richfitz/remake#170]
It would be nice to be able to use `package::function` syntax in commands. Currently, this fails, I'm guessing because the yaml parser is confused by the colons.

Example:

This works fine...

```yaml
packages:
    - readr

targets:
    all: 
        depends: myiris

    iris.csv:
        command: write.csv(I(iris), I('iris.csv'), row.names = FALSE)

    myiris:
        command: read_csv('iris.csv')
```

...but if I swap out the last line with `command: readr::read_csv('iris.csv')`, it fails with the following error:

```
Error in check_command_rule(command[[1]]) :
  While processing target 'myiris':
    Rule must be a character or name
Calls: <Anonymous> ... parse_target_command -> parse_command -> check_command_rule
Execution halted
```

**Comment by wlandau** [comment]
@ashiklom Agreed. You may want to look at the thread at #119 and the mention at #124.

**Comment by ashiklom** [comment]
Whoops! I somehow missed those when looking through the issues. In that case, this can be closed as a duplicate of #119.

**Comment by krlmlr** [comment]
Closing as duplicate.
